c10/util/Registry.h

```diff
@@ -126,92 +125,83 @@ class CAFFE2_API Registerer {
 
   template <class DerivedType>
   static ObjectPtrType DefaultCreator(Args... args) {
-    // TODO(jiayq): old versions of NVCC does not handle make_unique well
-    // so we are forced to use a unique_ptr constructor here. Check if it is
-    // fine to use make_unique in the future.
-    // return make_unique<DerivedType>(args...);
```

Comment:
So, it's not fine to use `make_unique`?

Reply:
Good catch, this is an old comment folks forgot to remove. Background info is, registry always uses std::unique_ptr back then, and nvcc does not allow make_unique, and we had to not use it. 

Today the registry returns PtrType which could be std::shared_ptr, and as a result the make_unique function cannot be used anyway, so this comment is moot...

Reply:
Good catch, this is an old comment folks forgot to remove. Background info is, registry always uses std::unique_ptr back then, and nvcc does not allow make_unique, and we had to not use it. 

Today the registry returns PtrType which could be std::shared_ptr, and as a result the make_unique function cannot be used anyway, so this comment is moot...

Reply:
(and this comment is deleted from the c10 registry header)